File: Wells/Source/Graphics/PremadeSeriesInfoCollection.vb
Public Class PremadeSeriesInfoCollection
    Property Title As String
    Property Values As List(Of PremadeSeriesInfo)

    Sub New()
        Values = New List(Of PremadeSeriesInfo)
    End Sub

    Sub Add(item As PremadeSeriesInfo)
        Values.Add(item)
    End Sub

    Sub Remove(item As PremadeSeriesInfo)
        Values.Remove(item)
    End Sub

    Public Overrides Function ToString() As String
        Return Title
    End Function
End Class

File: Wells/src/Graphics/PremadeSeriesInfoCollection.cs
using System.Collections.Generic;

namespace Wells.View.Graphics
{
    public class PremadeSeriesInfoCollection
    {
        public string Title { get; set; }
        public List<PremadeSeriesInfo> Values { get; set; }

        public PremadeSeriesInfoCollection() { Values = new List<PremadeSeriesInfo>(); }

        public void Add(PremadeSeriesInfo item)
        {
            Values.Add(item);
        }

        public void Remove(PremadeSeriesInfo item)
        {
            Values.Remove(item);
        }

        public override string ToString()
        {
            return Title;
        }
    }
}
